package io.openleap.authorization.exception;

public class FailedConnectionException extends RuntimeException {


  public FailedConnectionException(Throwable cause) {
    super(cause);
  }

}
